examples/test-controls/index.html

```diff
@@ -11,7 +11,7 @@
     <a-scene stats="true">
       <!-- We wrap the camera in an <a-entity> so we can position it without colliding with the HMD updates -->
       <a-entity>
-        <a-entity position="0 1.8 4" camera look-controls wasd-controls></a-entity>
+        <a-entity position="0 1.8 4" camera controls></a-entity>
```

Comment:
Documentation will help. If `controls` is used without anything else, it will use the default controls scheme. It'll also inject the other components.

People are still free to do ```controls="position: hmd-controls keyboard-controls gamepad-controls; rotation: hmd-controls touch-controls```, but it's convenient to abstract that if they want to use it out of the box.